### ISSUE TITLE
Add support for multiple controllers

### ISSUE THREAD

**Issue by chrisipa** [rock88/moonlight-nx#25]
Hi @rock88,

as far as I know other Moonlight clients (e.g. for Android), support up to 4 controllers for local multiplayer sessions.

Your version only seems to support 1 controller at the moment. 

Best regards
Chris

**Comment by moggencsscs** [comment]
Yes! @rock88 , this is a great product (much better than SkyNX, the only alternative) but this is one feature it sorely lacks. I'd pay you for the feature honestly

**Comment by manaboy** [comment]
I agree however I think the version of moonlight that this is based on doesn't support multiple controllers, hope for a future update that add this support.


Thanks

**Comment by symphonic5** [comment]
I know this is old, but it would really make this project perfect! I would love to play multiplayer games.

**Comment by manaboy** [comment]
It's a long process but you could install Lineage OS on the switch, then install moonlight for Android and it don't suffer from the lockup issues and supports multiple controllers. To install Lineage OS (Android 10) it requires you to repartition your microSD card, (remember to backup your emuMMC, make sure it says SD by clicking the bottom SDemummc on button otherwise you'll only be backing up your nand and not. SD emuMMC).

Linage OS also enables all the android emulators to work, this includes Dolphin Wii/GC so worth it.
